datahub-web-react/src/app/search/EntityGroupSearchResults.tsx

```diff
@@ -11,11 +12,20 @@ const styles = {
     header: { marginBottom: 20 },
     resultHeaderCardBody: { padding: '16px 24px' },
     resultHeaderCard: { right: '52px', top: '-40px', position: 'absolute' },
-    resultList: { width: '100%', borderColor: '#f0f0f0', marginTop: '8px', padding: '16px 48px' },
     seeAllButton: { fontSize: 18 },
     resultsContainer: { width: '100%', padding: '40px 132px' },
 };
 
+const ResultList = styled(List)`
+    &&& {
+        width: 100%;
+        border-color: ${(props) => props.theme.styles['border-color-base']};
+        margin-top: 8px;
+        padding: 16px 48px;
+        box-shadow: ${(props) => props.theme.styles['box-shadow']};
+    }
+`;
```

Comment:
since this is duplicated twice, you could make it its own component and import it into both search result pages